.travis.yml

```diff
@@ -1,4 +1,9 @@
-language: c
-compiler: gcc
```

Comment:
I don't know much about travis, but don't we still want to specify the compiler for C code?